modules/apps/transfer/module.go

```diff
@@ -213,6 +214,10 @@ func ValidateTransferChannelParams(
 		return sdkerrors.Wrapf(porttypes.ErrInvalidPort, "invalid port: %s, expected %s", portID, boundPort)
 	}
 
+	if portID != counterparty.PortId {
+		return sdkerrors.Wrapf(porttypes.ErrInvalidPort, "portID: %s does not match counterparty portID: %s", portID, counterparty.PortId)
+	}
```

Comment:
why is this code not included in main? We should open a separate pr with any changes made outside of `29-fee`